src/App.tsx

```diff
@@ -48,13 +55,43 @@ const App = (): JSX.Element => {
   useEffect(() => {
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
-    window.toggleDevtools = () => setShowDevtools((old) => !old);
+    window.toggleDevtools =
```

Comment:
It might be nice to explicitly say (in the commit message) something to the effect that the "new" devtools are accessed the same way as the "old" react-query ones? (At least, I had forgotten how to access the tools prior to this pr!)

Reply:
I suppose it isn't obvious if you've only explored this via the PR deployment, rather than locally, but in development builds there's also a button in the top-right of the UI, similarly to before:

<img width="270" alt="image" src="https://github.com/hackworthltd/primer-app/assets/12991097/ae3453d7-8a16-4f87-9e56-19137e5220d0">

Reply:
I had also forgotten how to enable this! Just a mention in the commit message would be fine.